### PR TITLE
Fix event listener

### PR DESCRIPTION
**Pull request by eimantaaas**
Fix event listener due to Oh Dear notification naming changes;
Fatal fix - Too few arguments to function Spatie\OhDearUptimeTile\OhDearWebhooksEventSubscriber::onUptimeCheckFailed(), 1 passed in ... and exactly 2 expected

### CODE REVIEW

File: src/OhDearWebhooksEventSubscriber.php [hunk 1]
```diff
@@ -8,32 +8,30 @@
 class OhDearWebhooksEventSubscriber
 {
     public function onUptimeCheckFailed(
-        OhDearWebhookCall $ohDearWebhookCall,
-        OhDearUptimeStore $ohDearUptimeStore
+        OhDearWebhookCall $ohDearWebhookCall
```
Comment (freekmurze) [inline src/OhDearWebhooksEventSubscriber.php:11]: Why did you refactor away the injection of the store?

If this is not really needed, please revert these changes.

**Review by freekmurze — CHANGES_REQUESTED** [review]
Thanks for this! I don't think all changes are necessary though.

File: src/OhDearWebhooksEventSubscriber.php (hunk 1 above)
Comment (eimantaaas) [reply src/OhDearWebhooksEventSubscriber.php:11]: Without that changes getting the mentioned fatal error, so that's why I refactored that. You can see details here: https://flareapp.io/share/Lm8E4VJP#F53
